File: 35- বৃত্তের বাইরে.c
#include<stdio.h>
#include<math.h>
int main()
{

    int t;
    scanf("%d",&t);
    while(t--)
    {

        double xc,yc,x,y,r;
        scanf("%lf%lf%lf%lf%lf",&xc,&yc,&r,&x,&y);

        double z = sqrt((pow((x-xc),2)+pow((y-yc),2)));

        if(z>r)
            printf("The point is not inside the circle\n");
        else
            printf("The point is inside the circle\n");
    }
    return 0;
}

File: 35- বৃত্তের বাইরে.py
t = int(input())

import math

while t!=0:
    a, b = input().split()
    a = float(a)
    b = float(b)
    r = float(input())
    x, y = input().split()
    x = float(x)
    y = float(y)

    d = math.sqrt(pow((x-a), 2)+pow((y-b), 2))

    if d>r:
        print("The point is not inside the circle")
    else:
        print("The point is inside the circle")
    
    t-=1
